tests/installation/grub_test.pm

```diff
@@ -80,9 +73,21 @@ sub run() {
         }
         send_key 'ret';
     }
-    if (get_var("XEN")) {
+    elsif (get_var("XEN")) {
         send_key_until_needlematch("bootmenu-xen-kernel", 'down', 10, 5);
     }
+    elsif (check_var('ARCH', 'x86_64') && check_var('DISTRI', 'sle')) {
+        record_soft_failure "bsc#995310 - Gnome fails to start";
+        send_key 'e';
+        # Move to end of kernel boot parameters line
+        for (1 .. 14) { send_key "down" }
```

Comment:
we run this test on so many scenarios and grub configs can be very different. So hardcoding 14 and 28 is an interesting approach - some assertions in between are needed.